crates/shared/src/sources/balancer_v2/swap.rs

```diff
@@ -573,7 +577,8 @@ mod tests {
         let tusd = H160::from_low_u64_be(3);
         let tokens = vec![dai, usdc, tusd];
         let scaling_exps = vec![0, 12, 12];
-        let amplification_parameter = AmplificationParameter::new(570.into(), 1000.into()).unwrap();
+        let amplification_parameter =
+            AmplificationParameter::new(570000.into(), 1000.into()).unwrap();
```

Comment:
THis value was just wrong... 
<img width="504" alt="image" src="https://github.com/cowprotocol/services/assets/4210206/ada56c9b-1635-4624-a086-305bf4b6e8ba">

Reply:
The test passed because of the incorrect `as_uint256` implementation on `AmplificationParameter`